metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorUtilSpec.groovy

```diff
@@ -42,4 +45,19 @@ class HiveConnectorUtilSpec extends Specification{
         '1'         | 1L
         '010m'      | 600L
     }
+
+    def "Test for check common view" () {
+        given:
+        def tableInfo = TableInfo.builder().metadata(metadata).build()
+        def isView = HiveTableUtil.isCommonView(tableInfo)
+        def isView2 = HiveTableUtil.isCommonView(tableInfo)
+        expect:
+        isView == output
+        isView2 == output
+        where:
+        metadata     | output
+        ['common_view' : "true"]    | true
+        ['common_view' : "false"]    | false
+        ['common_view_1' : "false"]    | false
```

Comment:
I'd like to add checks for the metadata `[:]`.  Also if metadata is nullable (I haven't checked) we should add the `null` case here as well.

Reply:
Added more cases.